app/src/modules/win.js

```diff
@@ -29,7 +29,7 @@ const win = {
     });
```

Comment:
NRTTL:
Switching destination application (first time switching to that application, consecutive switches preserves the amount) nullifies the sending amount.